Test/Tests/Integration/NodeManager.Tests.ps1

```diff
@@ -85,7 +85,7 @@ function Start-NodeMgr {
         Justification="Analyzer doesn't understand relation of Pester blocks"
         )]
         $NodeMgrJob = Start-Job -ScriptBlock {
-            contrail-nodemgr --nodetype contrail-vrouter
+            contrail-nodemgr --nodetype contrail-vrouter # allow unknown-functions(contrail-nodemgr)
```

Comment:
I think it's better to add `contrail-nodemgr` to `extras` in `shelly.toml` to leave the code cleaner